<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>get_ingredients</class>
 <widget class="QDialog" name="get_ingredients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>874</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_ingredients_list">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Select Ingredient</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Select Ingredient</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>------Vegetables------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Onion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Garlic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tomato (Buffalo)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tomato(Canned)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tomato (Cherry)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sun Dried Tomato</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pepper (Bell)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pepper (Sweet)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Basil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Parsley</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Broccoli</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Corn/ Sweet Corn</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spinach</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mushroom (Flat Cap)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mushroom (Portobello)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mushroom (White Button)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mushroom (Crimino)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mushroom (Shiitake)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mushroom (Oyster)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green Beans</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ginger</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chilli Pepper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rosemary</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Onion (White)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Onion (Red)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cucumber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sweet Potato</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pickle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Avocado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zucchini</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coriander/ Cilantro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mixed Vegetables (Frozen)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Olives</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Asparagus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cabbage</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cauliflower</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dill</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kale</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pumpkin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Squash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mint</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Scallion/ Green Onion</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shallot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Eggplant</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Beetroot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Butternut Squash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Horseradish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Leek</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Caper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brussle Sprout</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Artichoke</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Artichoke Heart</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chia Seed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Radish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Parsnip</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fennel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Turnip</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cress (Water)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cress (Garden)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rocket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bamboo Shoot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Celery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Celery Root</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>------Dairy------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Butter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Milk (Semi Skimmed)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Milk (Whole)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Parmesan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cheddar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mozzarella</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yogurt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cream (Single)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cream (Double)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ice Cream</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ricotta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Goats Cheese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Creme Fraiche</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Custard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mascarpone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Halloumi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Edam</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>------Baking &amp; Grains------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rice (Brown)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rice (White)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pasta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flour (Plain)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flour (Self Rasing)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bread</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Baking Powder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Baking Soda</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bread Crumbs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cornstarch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rolled Oats</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oats</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Noodles (Egg)</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txt_ingredient_backup">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_find_your_ingredients">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>80</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Find Your Ingredient</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_creat_your_ingredients_list">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Your Ingredients List</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cant_find_ingredient">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>190</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cant Find Your Ingredient?</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_enter_it_here">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter It Here:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>70</y>
     <width>20</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_measurement">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Select Measurement Type</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Select Measurement Type</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grams (g)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kilo Grams (kg)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Milliliter (ml)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Liter (l)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Table Spoon (tbsp)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tea Spoon (tsp)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Number of Items</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_measurement_type">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>79</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Measurement Type</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>477</x>
     <y>70</y>
     <width>20</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_amount">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_add_extra_info_to_ingredient_1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Extra Information / Add&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>20</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_add_extra_info_to_ingredient_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Description To Item&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_ingredient_amount">
   <property name="geometry">
    <rect>
     <x>561</x>
     <y>110</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_ingredient_description">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_current_ingredients_list">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Ingredients List</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_ingredients_list">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>661</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Ingredient to list</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
